src/main/resources/db/changelog/init/15062021_create_vendor_table.yml

```diff
@@ -36,12 +36,6 @@ databaseChangeLog:
                   constraints:
                     nullable: false
                     unique: true
-              - column:
```

Comment:
We cannot modify already executed changelogs.
In order to delete this column we should create a new liquibase script.

**BUT!** Before we delete this column, we should link vendors with locations according to our changes in the data model. So, we should perform data migration.

Reply:
fixed it